addons/hr_skills/static/src/fields/resume_one2many.scss

```diff
@@ -16,12 +16,15 @@
             }
 
             &:before {
-                @include o-position-absolute(0, $left: ($o-hrs-timeline-dot-size * .5 + $o-hrs-timeline-entry-padding));
-                width: 1px;
+                @include o-position-absolute(0, $left: calc(#{$o-hrs-timeline-dot-size} * .5 + #{$o-horizontal-padding} - #{$border-width} ));
```

Comment:
Well done, but in general better to avoid `calc()`.
We introduced a new sass function `o-to-rem()`, you can use that instead